ietf/doc/migrations/send_notices.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.28 on 2022-12-05 14:37
```

Comment:
This needs to be renamed to be 0048_send_notices.py (Though i'd suggest 0048_longer_notify_field.py)